programs/vote/src/vote_state/mod.rs

```diff
@@ -556,7 +556,7 @@ impl VoteState {
         timestamp: UnixTimestamp,
     ) -> Result<(), VoteError> {
         if (slot < self.last_timestamp.slot || timestamp < self.last_timestamp.timestamp)
-            || ((slot == self.last_timestamp.slot || timestamp == self.last_timestamp.timestamp)
+            || (slot == self.last_timestamp.slot
```

Comment:
Does this not need gated?

Reply:
@mvines and I think no. Validators are only timestamping every 4500 slots. If the timestamp hasn't progressed in 4500 slots, there's something majorly wrong.

Reply:
Hopefully no one's using the ETH2 NTP servers :za

Reply:
No, there's nothing that prevents more frequent timestamps.
So I agree, technically, we'd be vulnerable to a consensus attack during the upgrade.